.github/workflows/development.yml

```diff
@@ -3,6 +3,9 @@ on:
   push:
     branches:
       - main
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: false
```

Comment:
For deploying we should wait for the currently executing action to finish rather than cancel mid-deploy and leave things in a strange state.